### PR TITLE
test cases CRUD and file json

### PR DESCRIPTION
**Pull request by shekellkan**
...

### CODE REVIEW

**Comment by shekellkan** [comment]
revision

File: tests/CRUD/apk.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+describe('CRUD TCs of APK Routers', function () {
+	before(function (done) {
+		this.timeout(config.timeOut);
+		process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
+		done();
+	});	
+
+	it('verify that the service not exists', function(done) {
+		var code = "ResourceNotFound";
+		var message = "/apk does not exist";
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/apk.js]: this message should be in a config file, the same with the code

File: tests/CRUD/apk.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+		roomManagerAPI
+			.get(apkEndPoint, function(err, res){							
+				expect(code).to.equal(res.body.code);
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/apk.js]: you can apply more expects??

File: tests/CRUD/public-key.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+//Miguel Angel Terceros Caballero
+
+var expect = require('chai').expect;
+//import libraries
+var init = require('../../init');
+var config = require(GLOBAL.initialDirectory+'/config/config.json');
+var endPoints = require(GLOBAL.initialDirectory+config.path.endPoints);
+var roomManagerAPI = require(GLOBAL.initialDirectory+config.path.roomManagerAPI);
+var publicKey = require(GLOBAL.initialDirectory+'/config/publicKey.json');
+
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/public-key.js]: delete

File: tests/CRUD/public-key.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	it('verify that the public-key contains an begin block, version, end block', function(done) {		  
+						 
+		roomManagerAPI
+			.get(publicKeyEndPoint, function(err, res){	
+				//getting substrings of public-key
+				var cadena = res.body.content;
+				//assersions begin, version, end block of the key
+				expect(publicKeyType).to.equal(res.body.type);
+				expect(publicKeyBegin).to.equal(cadena.substring(0,36));
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/public-key.js]: try to not use hard code (0,36)

File: tests/CRUD/public-key.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+				expect(publicKeyVersion).to.equal(cadena.substring(38,69));
+				expect(publicKeyEnd).to.equal(cadena.substring(cadena.length-36, cadena.length-2));
```
Comment (AlejandroJeanCarlo) [inline tests/CRUD/public-key.js]: same with this two lines

**Comment by AlejandroJeanCarlo** [comment]
please review the new observations

File: tests/CRUD/public-key.js (hunk 3 above)
Comment (shekellkan) [inline tests/CRUD/public-key.js]: que linea quieres que elimine???

File: tests/CRUD/apk.js (hunk 2 above)
Comment (shekellkan) [inline tests/CRUD/apk.js]: no solo tiene esos expects

**Comment by shekellkan** [comment]
new changes

File: config/config.json [hunk 6]
```diff
@@ -1,9 +1,9 @@
 {
-	"url" : "https://172.20.208.194:4040",
-	"timeOut" : 4000,
+	"url" : "https://172.20.208.250:4040",
+	"timeOut" : 10000,
```
Comment (AlejandroJeanCarlo) [inline config/config.json]: this config is for all the test really you need this time? please explain this to all the team

File: tests/Smoke/roomResources.js [hunk 7]
```diff
@@ -0,0 +1,142 @@
+//Services
+//smoke test from Joaquin Gonzales Mosquera
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/roomResources.js]: ????? this mean that you copy the joaquin´s code? please do a double check to all the code in this class

File: tests/Smoke/roomResources.js [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+				done();
+			});
+	});
+
+	it('GET /services/{:serviceId}/rooms/{:roomId}/resources/{:roomResourceId} returns 200',function (done){	
+		//associate
+		var associateEndPoint = url + '/rooms/'+idRoom+'/resources';
+		//associate resource to room
+		roomManagerAPI
+			.post(token,associateEndPoint,associateResource, function(err, res){
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/roomResources.js]: this should be in a before method

File: tests/Smoke/roomResources.js [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+			});
+
+	});
+
+	it('PUT /services/{:serviceId}/rooms/{:roomId}/resources/{:roomResourceId} returns 200', function (done) {
+
+		var associateEndPoint = url + '/rooms/'+idRoom+'/resources';
+		//associate resource to room
+		roomManagerAPI
+			.post(token,associateEndPoint,associateResource, function(err, res){
```
Comment (AlejandroJeanCarlo) [inline tests/Smoke/roomResources.js]: before method to

File: config/config.json (hunk 6 above)
Comment (shekellkan) [inline config/config.json]: I changed this line to my tests

**Comment by shekellkan** [comment]
correct the files roomResources and config.json

**Comment by shekellkan** [comment]
new changes in tests/crud/ (apk, public-key and new file roomResource)

File: config/config.json [hunk 10]
```diff
@@ -17,8 +17,18 @@
 		"meetingsAPI" : "/lib/meetingsAPI",
 		"util" : "/util/util",
 		"roomManagerAPI" : "/lib/roomManagerAPI",
-		"locationConfig":"/config/locations.json"
+
+		"apk" : "/config/apk.json",
+		"publicKey" : "/config/publicKey.json",
+		"roomResource" : "/config/roomResource.json",
+
```
Comment (AlejandroJeanCarlo) [inline config/config.json]: remove this line

**Pull request merged**